.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/python:1-3.11",
```

Comment:
powerdump module is compatible with python 3.8 because of browserpy that is used in GUI command. 
we have an open issue to solve this dependency on 3.8
for now, maybe we need a dev container per module?

Reply:
Good point, I actually missed the 3.8 dependency. Unless there is a reason to run 3.11,  I will just update this to be 3.8 (and more reason to run it in a container vs installed locally)

You could do multiple dev containers if you wish,  or just update the version in this one based on requirements (depbot will try to bump the version on it if you have that enabled jfyi)

Reply:
sounds great.
lets update it for 3.8 for now

Reply:
Ok - swapped to 3.8